easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.2.2-CrayIntel-17.08.eb

```diff
@@ -0,0 +1,40 @@
+# contributed by Guilherme Peretti Pezzi and Luca Marsella (CSCS)
+easyblock = "ConfigureMake"
+
+name = 'QuantumESPRESSO'
```

Comment:
The recipe looks like it is done for official QE-6.2.2. This is not the case, as it is downloaded from the user's private repository.

Reply:
@toxa81 I could not find `QuantumESPRESSO` release `6.2.2` in the official repository, so I have downloaded it from the GitHub repository of `GIPAW`. I can adapt the recipe to build `GIPAW` on top of a standard build of `QuantumESPRESSO`, although it will use a less recent release (latest available release on qe-forge is `6.2.1`).